ecs-cli/modules/cli/local/converter/converter.go

```diff
@@ -24,28 +24,12 @@ import (
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/arn"
```

Comment:
nitpick: Copyright for both `converter.go` and `converter_test.go` are outdated.

https://github.com/aws/amazon-ecs-cli/blame/38abec487c428ea3e2d794625a250ab6ebbc39e8/ecs-cli/modules/cli/local/converter/converter.go#L1

Reply:
I fixed these for my next PR :)